src/components/forms/CommentAdder.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useContext, useState } from "react";
+import { UserContext } from "../../context/UserContext";
+import { addNewComment } from "../../utils/api";
+
+export default function CommentAdder({
+  article_id,
+  articleCommentList,
+  setArticleCommentList,
+}) {
+  const { users } = useContext(UserContext);
```

Comment:
I like the use of context here!
It may be clearer to have a single user held in a UserContext